dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ mock==2.0.0 ; python_version == '2.7'
 pytest-cov>=2.4.0,==2.*
 vcrpy>=1.10.0,<=1.10.9
 vcrpy-unittest==0.1.6
+joblib~=0.11
```

Comment:
Why not require joblib for the client install? The parallel module is useless without it. What's the point?